src/Symfony/Component/Form/Extension/Core/Type/TelType.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Form\Extension\Core\Type;
+
+use Symfony\Component\Form\AbstractType;
+
+class TelType extends AbstractType
```

Comment:
For what it's worth: the [misd-service-development/phone-number-bundle](https://github.com/misd-service-development/phone-number-bundle) named it [PhoneNumberType](https://github.com/misd-service-development/phone-number-bundle/blob/master/Form/Type/PhoneNumberType.php) and I actually prefer it over `Tel(ephone)Type`, regardless of what is the html implementation behind.